src/hooks/item.ts

```diff
@@ -105,6 +105,20 @@ export default (queryClient: QueryClient, queryConfig: QueryClientConfig) => {
         ...defaultOptions,
       }),
 
+    useItems: (ids: UUID[]) =>
```

Comment:
Perhaps add some conditions in `enabled`. I'm mainly thinking about checking if all ids are defined. 
Additionally, add a todo comment here: I think it's possible to add some optimisation and avoid fetching objects if they are already active in cache, but we can tackle that later. Or if you think it slows down the process too much, feel free to add the optimisation directly.